models/public/mobilenet-v3-small-1.0-224-tf/model.yml

```diff
@@ -19,21 +19,16 @@ description: >-
   For details see paper <https://arxiv.org/abs/1905.02244>.
 task_type: classification
 files:
-  - name: v3-small_224_1.0_float.tgz
-    size: 113230261
-    sha256: e60ad9f450f892ae56f8b9122ffc750d88940d9e6b63f6dc70649179fb3a9065
-    source: https://storage.googleapis.com/mobilenet_v3/checkpoints/v3-small_224_1.0_float.tgz
-postprocessing:
-  - $type: unpack_archive
-    format: gztar
-    file: v3-small_224_1.0_float.tgz
+  - name: weights_mobilenet_v3_small_224_1.0_float.h5
+    size: 10734624
+    sha256: bb828f93ae32afb89c2df190a39c06deaa63ea1d95fec93bceb52264f5bf61e8
+    source: https://storage.googleapis.com/tensorflow/keras-applications/mobilenet_v3/weights_mobilenet_v3_small_224_1.0_float.h5
 model_optimizer_args:
   - --reverse_input_channels
   - --input_shape=[1,224,224,3]
-  - --input=input
-  - --mean_values=input[127.5,127.5,127.5]
-  - --scale_values=input[127.5]
-  - --output=MobilenetV3/Predictions/Softmax
-  - --input_model=$dl_dir/v3-small_224_1.0_float/v3-small_224_1.0_float.pb
+  - --input=input_1
+  - --output=StatefulPartitionedCall/MobilenetV3small/Predictions/Softmax
+  - --saved_model_dir=$dl_dir/mobilenet_v3_small_224_1.0_float.savedmodel
 framework: tf
+quantizable: yes
```

Comment:
as I know both small and large models required custom configuration for quantization, it is not enough just add yes.
please ask @l-bat about correct pot configuration or remove this status for now (we add it automatically near to release time)

Reply:
Yes, we need to add ignored scopes for small and large models

Reply:
I added `quantization.yml` for small and large models. @l-bat , could you please review these files